pkg/network/ebpf/c/tracer/stats.h

```diff
@@ -70,6 +70,7 @@ static __always_inline void update_protocol_classification_information(conn_tupl
     normalize_tuple(&conn_tuple_copy);
 
     protocol_stack_t *protocol_stack = bpf_map_lookup_elem(&connection_protocol, &conn_tuple_copy);
+    set_protocol_flag(protocol_stack, FLAG_NPM_ENABLED);
```

Comment:
I'm not following what is the meaning of "NPM ENABLED"

Reply:
`NPM_ENABLED` means that the protocol classification probes are activated